### PR TITLE
Media: Use patch-package to fix horizontal jitter in the media gallery

### PR DESCRIPTION
**Pull request by diegovar**
Patches in https://github.com/neptunian/react-photo-gallery/pull/184

Addresses #4311

### CODE REVIEW

File: patches/react-photo-gallery+8.0.0.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
+diff --git a/node_modules/react-photo-gallery/.DS_Store b/node_modules/react-photo-gallery/.DS_Store
+new file mode 100644
+index 0000000..904f134
+Binary files /dev/null and b/node_modules/react-photo-gallery/.DS_Store differ
+diff --git a/node_modules/react-photo-gallery/dist/react-photo-gallery.esm.js b/node_modules/react-photo-gallery/dist/react-photo-gallery.esm.js
```
Comment (swissspidy) [inline patches/react-photo-gallery+8.0.0.patch]:
```suggestion
diff --git a/node_modules/react-photo-gallery/dist/react-photo-gallery.esm.js b/node_modules/react-photo-gallery/dist/react-photo-gallery.esm.js
```

**Comment by swissspidy** [comment]
@cvolzke4 Can we revert #4363 with this?

**Comment by swissspidy** [comment]
@jhtin I assume #4378 is still needed?

**Comment by ezxyz** [comment]
@swissspidy Not sure if the persistent scrollbar would be needed after this fix: @diegovar could you clarify?

**Review by ezxyz — DISMISSED** [review]
Is it possible to test this?

**Comment by diegovar** [comment]
> @cvolzke4 Can we revert #4363 with this?

I believe that's unrelated.

**Comment by diegovar** [comment]
> @swissspidy Not sure if the persistent scrollbar would be needed after this fix: @diegovar could you clarify?

I believe it would.

**Comment by diegovar** [comment]
> Is it possible to test this?

I don't think it's easy or possible to test this, a it's a mitigation for a very specific edge case that's hard to reproduce on a separate library.

**Comment by cvolzke4** [comment]
> @cvolzke4 Can we revert #4363 with this?

Correct, though I couldn't see that there's any performance benefit to using pure components for PaginatedMediaGallery and/or MediaGallery (though it wouldn't hurt performance either).

**Review by swissspidy — APPROVED** [review]
